<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGravityVolumePanel</class>
 <widget class="QWidget" name="CGravityVolumePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>112</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="NUM_POINTS_LABEL">
     <property name="text">
      <string>Num Points: 23</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Shape Editing</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QEditorToolButton" name="EDIT_SHAPE">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="SELECTED_POINT">
        <property name="text">
         <string>Selected Point: no selection</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEditorToolButton</class>
   <extends>QPushButton</extends>
   <header>Controls/ToolButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
